### PR TITLE
add clickable logo on login page

### PR DESCRIPTION
**Pull request by orby-tech**
# Contribution checklist

## Brief description
Added clickable logo on login page with way to bold.ru
image

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: plugins/login-resources/src/components/LoginApp.svelte [hunk 1]
```diff
@@ -88,14 +88,15 @@
       />
     </picture>
 
-    <div
+    <a
+      href="https://bold.ru"
```
Comment (haiodo) [inline plugins/login-resources/src/components/LoginApp.svelte]: This should be configurable with environment variable, and passed from front service using config.

Comment (orby-tech) [reply plugins/login-resources/src/components/LoginApp.svelte]: Thanks! I added a variable to the config

File: dev/prod/src/platform.ts [hunk 2]
```diff
@@ -138,6 +139,7 @@ export async function configurePlatform() {
   const config: Config = await (await fetch(devConfig? '/config-dev.json' : '/config.json')).json()
   console.log('loading configuration', config)
   setMetadata(login.metadata.AccountsUrl, config.ACCOUNTS_URL)
+  setMetadata(login.metadata.LogoWayUrl, config.LOGO_WAY_URL)
```
Comment (haiodo) [inline dev/prod/src/platform.ts:139]: You need to update anticrm/server/front/src/index.ts as well, or variable will not work.

Comment (orby-tech) [reply dev/prod/src/platform.ts:139]: Great, thank you!

Added

File: plugins/login-resources/src/components/LoginApp.svelte (hunk 1 above)
Comment (orby-tech) [reply plugins/login-resources/src/components/LoginApp.svelte]: @haiodo can you help me please with uitests? 
Unfortunately, I don’t understand at all what the problem is with it

Comment (haiodo) [reply plugins/login-resources/src/components/LoginApp.svelte]: Please try to rebase at least.

Comment (haiodo) [reply plugins/login-resources/src/components/LoginApp.svelte]: And you need to pass LOGOWAY_URL variable to tests/docker-compose.yaml front container.

Comment (orby-tech) [reply plugins/login-resources/src/components/LoginApp.svelte]: Thank you
I'm added variable to compose and tried to to pull all changes but problem with pipeline is not solved

**Pull request closed without merging**